editor/editor_inspector.cpp

```diff
@@ -3230,83 +3183,48 @@ void EditorInspector::update_tree() {
 				classname = get_edited_object()->get_class();
 			}
 
-			// Search for the property description in the cache.
-			HashMap<StringName, HashMap<StringName, PropertyDocInfo>>::Iterator E = doc_info_cache.find(classname);
+			// Search for the doc path in the cache.
+			HashMap<StringName, HashMap<StringName, String>>::Iterator E = doc_path_cache.find(classname);
 			if (E) {
-				HashMap<StringName, PropertyDocInfo>::Iterator F = E->value.find(propname);
+				HashMap<StringName, String>::Iterator F = E->value.find(propname);
 				if (F) {
 					found = true;
-					doc_info = F->value;
+					doc_path = F->value;
 				}
 			}
 
 			if (!found) {
 				bool is_native_class = ClassDB::class_exists(classname);
-
-				// Build the property description String and add it to the cache.
-				DocTools *dd = EditorHelp::get_doc_data();
-				HashMap<String, DocData::ClassDoc>::ConstIterator F = dd->class_list.find(classname);
-				while (F && doc_info.description.is_empty()) {
```

Comment:
From my tests, removing this loop didn't affect anything negatively. But I would like a second opinion on this.

Reply:
Turns out it did. See https://github.com/godotengine/godot/pull/82051#discussion_r1341255720.